subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/SymlinkFileSystemWatchingIntegrationTest.groovy

```diff
@@ -142,53 +140,4 @@ class SymlinkFileSystemWatchingIntegrationTest extends AbstractFileSystemWatchin
         then:
         executedAndNotSkipped ":myTask"
     }
-
-    def "disable file system watching when trying to watch symlinked directory"() {
```

Comment:
Where did this go?

Reply:
We now can't watch a symlinked directory, since project directories are canonicalized. So this cannot happen any more. We need to re-add the test when we register other directories which could be watched.

Reply:
Shall we `@Ignore` the test instead then?

Reply:
I ignored the test. Let's not forget about it.